src/Divider/Divider.js

```diff
@@ -11,7 +11,7 @@ export const styleSheet = createStyleSheet('MuiDivider', (theme) => {
   return {
     root: {
       height: 1,
-      margin: '0 -1px 0 0',
+      margin: 0,
```

Comment:
@kof It seems counter-intuitive to me than a divider that a divider uses no space. At the office we have `margin: -1px 0 0;`.
I'm not completely sure around what I'm doing, but it's not breaking any visual regressions tests. So, I'm assuming it's fine.